packages/service-library/tests/fastapi/test_rabbitmq.py

```diff
@@ -0,0 +1,159 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+import asyncio
+from collections.abc import Callable, Mapping
+from typing import Any
+
+import aiodocker
+import pytest
+from faker import Faker
+from fastapi import FastAPI
+from models_library.rabbitmq_messages import LoggerRabbitMessage, RabbitMessageBase
+from pytest_mock.plugin import MockerFixture
+from servicelib.fastapi.rabbitmq import get_rabbitmq_client
+from servicelib.rabbitmq import BIND_TO_ALL_TOPICS, RabbitMQClient
+from settings_library.rabbit import RabbitSettings
+from simcore_service_api_server.services.rabbitmq import post_message
```

Comment:
Yes, this is wrong. These to imports from `simcore_servcie_*` should not go here. Probably the auto-import introduced this problem

Reply:
Yep, I will fix this one